docs/references/express/overview.mdx

```diff
@@ -0,0 +1,165 @@
+---
+title: Clerk Express SDK
+description: Learn how to integrate Express into your Clerk application.
+---
+
+Clerk makes it simple to add authentication to your Express application. This documentation covers the capabilities and methods available from Clerk's Express SDK.
+
+See the [quickstart](/docs/quickstarts/express) to get started.
+
+## Available methods
+
+### `clerkMiddleware()`
+
+The `clerkMiddleware()` helper integrates Clerk authentication into your Express application. It is required to be set in the middleware chain before using other Clerk utilities, such as `requireAuth` and `getAuth()`.
```

Comment:
I do feel like we should perhaps explain what it actually does here to clarify, which is just running `authenticateRequest` on the req object - so effectively, checking the cookies and headers to look for a session JWT, and if it finds one in either place, putting it on the request object under the `auth` key.

Reply:
should we update our canonical clerkmiddleware reference then (the nextjs one)??